### DISCUSSION TITLE
可以左右分栏吗

### DISCUSSION THREAD

**Discussion by sofacoder35** [Keldos-Li/typora-latex-theme#94] [Q&A] [answered]
是否可以左右分栏呢

**Comment by RalXYZ** [comment]
这个有点难😭远期规划里可以加一下这个

**Comment by Keldos-Li** [comment, marked as answer]
抱歉，不可以的。

左右分栏是单页面逻辑，但是markdown是长页面逻辑。分栏是可以分栏（事实上这个代码也已经被注释在css中了），但结果上是，网页上就是左边一栏，右边一栏，打印之后并不能**按打印页**重新分栏，会造成左栏从第一页到最后一页，再起头右栏从第一页到最后一页……这不是我们想要的结果。

我们之前也考虑过您的这个想法，最后无奈发现在现行框架下无法解决。再次感谢您的建议。
